toolkit/cli/smart-contracts-commands/src/governance.rs

```diff
@@ -28,7 +28,7 @@ pub struct InitGovernanceCmd {
 	#[clap(flatten)]
 	common_arguments: crate::CommonArguments,
 	/// Governance authority hash to be set.
-	#[arg(long, short = 'g')]
+	#[arg(short, long, num_args = 1.., value_delimiter = ' ')]
```

Comment:
I propose updating the help message above to simply "Governance authority to be set". If you prefer verbose then "Multisig governance authorities keys hashes to be set"